scripts/exgfs_wave_post_pnt.sh

```diff
@@ -393,14 +393,14 @@ source "${USHgfs}/preamble.sh"
     then
       export dtspec=3600.
       # Construct the wave_outp_spec (spec) command to run on each buoy in buoy_lst.txt
-      sed "s/^\(.*\)$/${escaped_USHgfs}\/wave_outp_spec.sh \1 ${ymdh} spec ${escaped_SPECDATA} > ${escaped_SPECDATA}\/spec_\1.out 2>\&1/" buoy_lst.txt >> "tmpcmdfile.${FH3}"
+      sed "s?^\(.*\)$/${escaped_USHgfs}?wave_outp_spec.sh \1 ${ymdh} spec ${escaped_SPECDATA} > ${escaped_SPECDATA}\/spec_\1.out 2>\&1?" buoy_lst.txt >> "tmpcmdfile.${FH3}"
```

Comment:
I'm not sure why you are changing this script at all, but if you are going to change the separation character, please pick one that _isn't_ a special regex character. Please revert.

Reply:
AWS does not like "/" delimiter mixed with "\/".
Which one do you like?
Will "|" be better.

Reply:
Pipe (`|`) is also a reserved character for regex (OR). `%` or `@` would be better, as they are neither reserved by regex nor commonly used in paths. A non-standard character like `•` (U+00B7) would be better yet. Ultimately though, I want to know why it should work differently on AWS. Bash is bash.

If you do need to change the separator, also please take care you are doing it correctly. Here you have replaced  what was intended to be a literal `/` (not a separator) with a separator.

CC @aerorahul

Reply:
There was one time, I saw error with sed. But I could not repeat it.
So I revert the changes for now, and if the errors come back, I'll report and try '%' as delimiter.

I see some "wave" problem. will add as commit.